Assets/Scripts/BulletManagement/SlipTimeEmitter.cs

```diff
@@ -11,14 +11,100 @@ public class SlipTimeEmitter : EmitterBase, ISlipTimeAdherent
         [SerializeField]
         private SlipTimeManager slipTimeManager;
         
+        // Running total of emitted bullet waves
+        private int cyclesEmitted = 0;
+
+        private bool limited = false;
+
+        public float bulletSpeedMultiplier = 1.0f;
```

Comment:
Same here for this variable, duplicate variable, good to remove since base class already has it public.

Reply:
Whoops, yeah, this totally shouldn't be here.